client/client_linux.go

```diff
@@ -107,8 +107,23 @@ type LinuxConfigFsQuoteProvider struct{}
 
 // IsSupported checks if TSM client can be created to use ConfigFS system.
```

Comment:
Could you update the comment?

Reply:
Done. I found that tdx-guest is not the provider name despite documentation, but actually tdx_guest\n, so I'm finishing some e2e tests internally before merging.